Cython/Compiler/FusedNode.py

```diff
@@ -142,6 +142,7 @@ def copy_cdef(self, env):
             # Make the types in our CFuncType specific.
             try:
                 type = copied_node.type.specialize(fused_to_specific)
+                entry_type = copied_node.entry.type.specialize(fused_to_specific)
```

Comment:
This is ugly but I need to ensure that final entry will have type with base class instead of child one - `double (A, double) except *` instead of `double (B, double) except *`. This is because `copied_node.type` is different than `copied_node.entry.type` !!!!!!

This is needed to generate properly cast in following line of C code:

```c
  __pyx_vtable_11fused_error_B.__pyx_base.__pyx_fuse_0f = (double (*)(struct __pyx_obj_11fused_error_A *, double))__pyx_fuse_0__pyx_f_11fused_error_1B_f;
```

I am not sure if it is correct that node type is different than entry type so not sure how to proceed with it...